cypress/integration/common/simpleSelect.feature

```diff
@@ -48,6 +48,8 @@ Feature: Select component
       | key       | position | value  |
       | downarrow | first    | Amber  |
       | uparrow   | eleventh | Yellow |
+      | pagedown  | fourth   | Brown  |
```

Comment:
Where did we get to with updating this Cypress test? It needs to assert that the correct option is focused, not just that the select list is open.